docs/changelog.rst

```diff
@@ -3,6 +3,22 @@ Changelog
 
 .. scriv-insert-here
 
+.. _changelog-1.0.8:
+
+funcx & funcx-endpoint v1.0.8
+-----------------------------
+
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- The endpoint can now register metadata such as IP, hostname, and configuration values
+  with the funcX services.
```

Comment:
Does it make sense to reword this if we aren't releasing the services alongside this, since users won't be able to retrieve this information?

Reply:
I don't think this wording change would be a blocker.

Reply:
A more complete and 'correct' explanation about what timeframe the retrieval would work is a bit more complicated, and only applies for a week or so.  I'm leaning toward it's fine to leave it, and if anyone asks we can explain individually?  (As it's not a feature people are looking forward to using right away).

Reply:
I concur with your analysis, Lei.  The _worst_ that happens is that an advanced user asks us a question and we respond individually.

Reply:
I'm convinced 👍